product_sticker/views/product_attribute_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="product_attribute_view_form" model="ir.ui.view">
+        <field name="name">product.attribute.sticker.form</field>
+        <field name="model">product.attribute</field>
+        <field name="inherit_id" ref="product.product_attribute_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button
+                    name="%(action_product_sticker)d"
+                    type="action"
+                    class="oe_stat_button"
+                    icon="fa-picture-o"
+                    attrs="{'invisible': [('id', '=', False)]}"
```

Comment:
This domain is strange... I'm curious, is it needed?

Reply:
It's for the context below, but can be removed with an `oe_read_only` class